site/2016/call-for-tutorials.md

```diff
@@ -6,7 +6,9 @@
 Commercial Users of Functional Programming (CUFP) is an annual meeting
 co-located with the International Conference on Functional Programming which
 this year will take place in Nara, Japan from the 22nd to 24th of September
-2016. CUFP aims to bridge the gap between academia and users applying functional
+2016.
+
+CUFP aims to bridge the gap between academia and users applying functional
```

Comment:
For some reason this is being rendered as a list on the CFT page, I split this paragraph in a newline